### PR TITLE
WEB-433: Add visiting scholar program

### PR DESCRIPTION
**Pull request by lscheinkman**
@brev Please review.
I've put the tests in a separated commit so it is easier to review.

You can check the results on my forks's gh-pages: https://lscheinkman.github.io/numenta-web/

CC: @rhyolight

### CODE REVIEW

File: packages/numenta.com/pages/careers-and-team/careers/visiting-scholar-program.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    pathways
+
+-   Cortico-thalamic connections
+
+-   Neural representation and the cortical code, specifically sparse
+    distributed representations
+
+-   Representations of action and movement in motor cortex
+
+Our recent publications can be seen at http://www.numenta.com/papers
```
Comment (brev) [inline packages/numenta.com/pages/careers-and-team/careers/visiting-scholar-program.md]: @lscheinkman please use URL form: http://numenta.com/papers/ thanks

Comment (lscheinkman) [inline packages/numenta.com/pages/careers-and-team/careers/visiting-scholar-program.md]: @brev I will update the link. 
BTW, this page was automatically generated from the `.docx` via `pandoc`: 
```
pandoc -o visiting-scholar-program.md visiting-scholar-program.docx
```
I just had to add the header metadata part, everything else came directly from the word document and the tools integrated with the site.

🥇 Great job on the site! It is really easy to understand and update.

Comment (brev) [inline packages/numenta.com/pages/careers-and-team/careers/visiting-scholar-program.md]: r.e. `pandoc` wow thanks 😂 that will save me so much time in the future

thank you sir!

File: packages/numenta.com/pages/careers-and-team/careers/visiting-scholar-program.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ interest:
 
 -   Representations of action and movement in motor cortex
 
-Our recent publications can be seen at http://www.numenta.com/papers
+Our recent publications can be seen at http://numenta.com/papers
```
Comment (brev) [inline packages/numenta.com/pages/careers-and-team/careers/visiting-scholar-program.md]: sorry, the end slash is the most important part :)  http://numenta.com/papers/

**Comment by brev** [comment]
thanks

File: packages/numenta.com/pages/careers-and-team/careers/visiting-scholar-program.md [hunk 3]
```diff
@@ -26,11 +26,10 @@ several months.  While we expect the visitor to be salaried through
...
 
-The environment at Numenta is highly collaborative, energetic, and open.
- Our research is fully transparent. Our source code, including daily
-research code commits, is available via our open source Github project
-NuPIC. We are located in the heart of Silicon Valley (Redwood City, CA),
-close to Stanford University and San Francisco. 
+The environment at Numenta is highly collaborative and energetic. We support
+Open Science. All our source code is available via our open source Github
+project NuPIC. We are located in the heart of Silicon Valley (Redwood City, CA),
+close to Stanford University and San Francisco..
```
Comment (brev) [inline packages/numenta.com/pages/careers-and-team/careers/visiting-scholar-program.md]: just double-checking, should it be two periods at the end? ".."
thx

**Pull request merged**